### DISCUSSION TITLE
Error/Warning when trying to export const variables

### DISCUSSION THREAD

**Discussion by Clayrok** [godotengine/godot-proposals#7779] [Scripting]
Hi,

First experience with Godot. I've heard about the Export attribute so I tried adding some over the auto generated "Speed" and "JumpVelocity" variables on a C# script inheriting from CharacterBody3D.
But after returning to the engine I wasn't able to see those variables in the inspector.
I've tried to rebuild the solution and restart the engine but still nothing.
I've finally read my code again and found out those variables are marked as const.

It would be nice to get an error or at least a warning when trying to export const values.

**Comment by Mickeon** [comment]
GDScript gives an error appropriately for this situation, so this is more like asking for a bug-fix rather than a feature.

**Reply by Clayrok** [reply]
Oh, good to know. I'll open an issue then. Thanks
